model/riscv_types.sail

```diff
@@ -389,6 +389,8 @@ enum sopw = {RISCV_SLLIW, RISCV_SRLIW,
 enum amoop = {AMOSWAP, AMOADD, AMOXOR, AMOAND, AMOOR,
               AMOMIN, AMOMAX, AMOMINU, AMOMAXU}   /* AMO ops */
 enum csrop = {CSRRW, CSRRS, CSRRC}                /* CSR ops */
+enum cbop = {CBO_CLEAN, CBO_FLUSH, CBO_INVAL, CBO_ZERO}  /* cbo ops*/
+enum preop = {PREFETCH_I, PREFETCH_R, PREFETCH_W} /* prefetch ops*/
```

Comment:
Follow the convention below; put them in a separate block and include the extension names